src/Analysis/AnalysisApp.jsx

```diff
@@ -138,7 +138,9 @@ class AnalysisApp extends React.Component {
     case 'AtlasDataDictionary': {
       return (
         <div className='analysis-app_flex_row'>
-          <AtlasDataDictionaryContainer />
+          <AtlasDataDictionaryContainer
+            title={analysisApps[app].dataDictionaryVersion}
```

Comment:
This is the wrong prop name. I believe this needs to be `dataDictionaryVersion`. I was unable to get this to work locally with `title` as the prop name.